src/App.vue

```diff
@@ -11,7 +11,7 @@ import Header from './components/Header.vue';
 import Nav from './components/Nav.vue';
 
 export default {
-  name: 'Rancher Desktop',
+  name: 'App',
```

Comment:
Shouldn't the name still be `Rancher Desktop`?

Reply:
That's the _element_ name, i.e. `<App>…</App>`; using it as `<Rancher Desktop>` wouldn't work.  That's why I introduced a new `title` property instead.  Actually, I only changed it because Vue started complaining; I assume that was an API change between Vue 2 and Vue 3.

Reply:
Ah, gotcha.